src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +271,52 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool IsAccessorListWithoutAccessorsWithBlockBody(SyntaxNode? node)
+            => node is AccessorListSyntax accessorList &&
+                accessorList.Accessors.All(a => a.Body == null);
+
+        private static bool HasInitializer(BasePropertyDeclarationSyntax basePropertyDeclaration)
+            => basePropertyDeclaration is PropertyDeclarationSyntax property && property.Initializer != null;
+
+        private static bool IsAccessorListFollowedByInitializer(SyntaxNode? node)
+            => node is AccessorListSyntax accessorList &&
+                accessorList.Parent is BasePropertyDeclarationSyntax basePropertyDeclarationSyntax &&
+                HasInitializer(basePropertyDeclarationSyntax);
```

Comment:
this is the only place i can see where you call HasInitializer.  But you first type-check to BaseProperty, and then type-check to Property in the actual HasInitializer.  It seems like we could change this code to be: